docs/core-services/container-orchestration-provider-usage.md

```diff
@@ -84,6 +84,8 @@ To begin configuring the container, look under **Services** and select the item
 
 - **Restart Container On Failure** - A boolean that tells the container engine to automatically restart the container when it has failed or shut down.
 
+- **Container Image Enforcement Digest** - A string representing the image digest allowed to be run from this container instances. It is used in the [Container Enforcement](#container-enforcement) service provided by the Container Orchestration Service.
```

Comment:
```suggestion
- **Container Image Enforcement Digest** - A string representing the digest for the image that will be allowed to run by this container instance. It is used in the [Container Enforcement](#container-enforcement) service provided by the Container Orchestration Service.
```

Maybe it is worth to provide an example of the string? Do we expect this to follow a certain pattern?